mne/preprocessing/tests/test_ica.py

```diff
@@ -130,6 +130,33 @@ def test_ica_rank_reduction():
 
 
 @requires_sklearn
+def test_ica_reset():
+    """Test ICA resetting"""
+    raw = io.Raw(raw_fname, preload=True).crop(0, stop, False).crop(0.5)
```

Comment:
I don't like it either.

@Eric89GXL can you take over if @dengemann is busy?

thx